CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 0.4.2-dev
 
+### Fixed
+
+- Incorrect default config path in `--help` on Windows and macOS
```

Comment:
Can we get this patch into v0.4.1?

Reply:
Honestly I think I'd like to put this into 0.4.2, since this isn't really a bug that is very time sensitive and it has been around for a long, long time anyways. So extending the scope of the 0.4.1 release for this after the RC is already out seem strange.

If there were other bugs that required another RC anyways, I wouldn't mind as much, but I think the RC is actually looking pretty good as-is.

Reply:
Thoughts @nixpulvis?

Reply:
I don't have a strong opinion either way. I think it's fine to wait, I'm generally a fan of keeping releases contained.